recaptcha_enterprise/demosite/app/static/demo-6df0841a.js

```diff
@@ -3723,10 +3743,19 @@ class RecaptchaDemo extends s {
     this._score = value;
     this.requestUpdate("score", oldValue);
     const buttonElement = document.querySelector("recaptcha-demo > button");
-    if (buttonElement && this._score && this.step !== "comment") {
-      window.setTimeout(() => {
-        buttonElement.innerText = "Go to next demo";
-      }, 100);
+    if (buttonElement && this._score) {
+      // TODO: redesign per https://b.corp.google.com/issues/278563766
```

Comment:
Use b/278563766 - although Buganizer is public, the link shown here is an internal link versus public-facing link (https://issuetracker.google.com/issues)
```suggestion
      // TODO: redesign per b/278563766
```

Reply:
Done.. thanks!